imperial_coldfront_plugin/settings.py

```diff
@@ -14,3 +14,16 @@
 GPFS_API_USERNAME = ENV.str("GPFS_API_USERNAME", default="")
 GPFS_API_PASSWORD = ENV.str("GPFS_API_PASSWORD", default="")
 GPFS_API_VERIFY = ENV.bool("GPFS_API_VERIFY", default=False)
+
+LDAP_USERNAME = ENV.str("LDAP_USERNAME", default="")
+LDAP_PASSWORD = ENV.str("LDAP_PASSWORD", default="")
+LDAP_URI = ENV.str("LDAP_URI", default="")
+LDAP_USER_OU = ENV.str(
```

Comment:
Probably, some of these environment variables with obscure names or purpose should have a minimal docstring explaining what they are about.